ui/src/app/component/item-selector/item-selector.component.ts

```diff
@@ -239,4 +238,10 @@ export class ItemSelectorComponent
     }
     return false;
   }
+
+  getAvailableTeams(): void {
+    this.availableGetter(this.selectedSubMenuItem).subscribe(
```

Comment:
👍 for "subMenu" as a generic name for these things. Do we also need to update `getAvailableTeams` and `availableGetter` to remove the teams reference? Maybe something like `getSubMenuList`?